CoreFoundation/Base.subproj/CFAvailability.h

```diff
@@ -89,6 +89,29 @@
 #define CF_DEPRECATED_IOS(_iosIntro, _iosDep, ...)
 #endif
 
+#if DEPLOYMENT_RUNTIME_SWIFT
+// PortableFoundation is a standalone module and does not depend on system version
+#undef CF_AVAILABLE
+#undef CF_AVAILABLE_MAC
+#undef CF_AVAILABLE_IOS
+#undef CF_DEPRECATED
+#undef CF_DEPRECATED_MAC
+#undef CF_DEPRECATED_IOS
+
```

Comment:
I am not certain this is the right move here. It has some consequences that might sign us all up for things that would tie our hands for fixing things in the future. It might be a good idea to break this up into more than one pull request so we can consider the full meaning individually